README.rst

```diff
@@ -395,6 +396,20 @@ Allows changing the default OpenWRT upgrader settings, eg:
   useful in case  the upgrade command hangs (it happens on older OpenWRT versions);
   defaults to ``90`` seconds
 
+``OPENWISP_FIRMWARE_IMAGE_BASEURL``
```

Comment:
The ability to change the baseurl is required for `docker-openwisp`, so that we can point to the `firmware.YOUR_DOMAIN.com` domain! :smile: